src/Selector/Xpath/Manipulator.php

```diff
@@ -44,7 +44,7 @@ public function prepend($xpath, $prefix)
         }
 
         // Split any unions into individual expressions.
-        foreach (preg_split(self::UNION_PATTERN, $xpath) as $expression) {
```

Comment:
it is used further above and for backwards compatibility reasons we should not remove it, other libraries might rely on it.